mood-analysis.rb

```diff
@@ -28,5 +34,62 @@ def analyze_mood(words)
   "05/11 Yay, yay, yay! I'm having a awfuly great day."
 ]
 
-puts analyze_mood(text[0])
-puts analyze_mood(text[1])
+# print out date + mood by calling analyze mood
+def week_moods(array)
+  array.each do |day|
+    date = day[0..4]
+    puts "#{date} #{analyze_mood(day)}"
+  end
+end
+
+def happy_days(array)
+  happy = 0
+  days = 0
+  array.each do |day|
+    mood = analyze_mood(day)[-3..-1]
+    if mood == ":-)"
+      happy += 1
+    end
+    days += 1
+    if happy == 3
+      break
+    end
+  end  
+  if happy == 3
+    puts "It takes #{days} entries for 3 happy days to occur"
+  else
+    puts "You are having quite a bad week."
+  end
+end
```

Comment:
I'd rather that this method returned just the fixnum of how many days rather than putting that inside a string. This would allow for us to do other things with that information instead of being tied to this specific use case.